src/bootstrap.tsx

```diff
@@ -699,12 +703,18 @@ const renderApp = async () => {
       await i18n.init(initOpts);
     }
 
+    let type: any = 'warning';
```

Comment:
Can potentially be typed via ` AlertProps['type']`.

Reply:
Nice, fixed in 069174b